src/RestServer/Controllers/v1/LedgerController.cs

```diff
@@ -0,0 +1,387 @@
+        /// <returns>An array of Signer object.</returns>
+        /// <response code="200">Successful</response>
+        /// <response code="400">An error occurred. See Response for details.</response>
+        [HttpGet("transactions/{hash:required}/signers", Name = "GetTransactionSigners")]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(Signer[]))]
+        public IActionResult GetTransactionSigners(
+            [FromRoute( Name = "hash")]
+            UInt256 hash)
+        {
+            if (NativeContract.Ledger.ContainsTransaction(_neosystem.StoreView, hash) == false)
+                throw new TransactionNotFoundException(hash);
+            var tx = NativeContract.Ledger.GetTransaction(_neosystem.StoreView, hash);
+            return Ok(tx.Signers);
+        }
+
+        /// <summary>
+        /// Gets the transaction attributes of a transaction.
+        /// </summary>
+        /// <param name="hash" example="0xad83d993ca2d9783ca86a000b39920c20508c8ccae7b7db11806646a4832bc50">Hash256</param>
+        /// <returns>An array of the transaction attributes object.</returns>
+        /// <response code="200">Successful</response>
+        /// <response code="400">An error occurred. See Response for details.</response>
+        [HttpGet("transactions/{hash:required}/attributes", Name = "GetTransactionAttributes")]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(TransactionAttribute[]))]
+        public IActionResult GetTransactionAttributes(
+            [FromRoute( Name = "hash")]
+            UInt256 hash)
+        {
+            if (NativeContract.Ledger.ContainsTransaction(_neosystem.StoreView, hash) == false)
+                throw new TransactionNotFoundException(hash);
+            var tx = NativeContract.Ledger.GetTransaction(_neosystem.StoreView, hash);
+            return Ok(tx.Attributes);
+        }
+
+        #endregion
+
+        #region Memory Pool
+
+        /// <summary>
+        /// Gets memory pool.
+        /// </summary>
+        /// <param name="skip" example="1">Page</param>
+        /// <param name="take" example="50">Page Size.</param>
+        /// <returns>An array of the Transaction object.</returns>
+        /// <response code="200">Successful</response>
+        /// <response code="400">An error occurred. See Response for details.</response>
+        [HttpGet("memorypool", Name = "GetMemoryPoolTransactions")]
+        [ProducesResponseType(StatusCodes.Status200OK, Type = typeof(Transaction[]))]
+        public IActionResult GetMemoryPool(
+            [FromQuery(Name = "page")]
+            int skip = 1,
+            [FromQuery(Name = "size")]
+            int take = 50)
+        {
+            if (skip < 0 || take < 0 || take > RestServerSettings.Current.MaxPageSize)
+                throw new InvalidParameterRangeException();
+            return Ok(_neosystem.MemPool.Skip((skip - 1) * take).Take(take));
```

Comment:
Memory pool can change from request to request, I'm not sure paging really fits here.

Reply:
Yes i thought about that too. But if we were as busy as `etherium` and blocks can have `10,000` transactions in it. I figure paging would be good.

Reply:
Vote up for paging removal, no practical usage for this option.